tests/positive/NamedArguments.juvix

```diff
@@ -89,5 +89,5 @@ module FakeRecord;
     mkPair (snd := unit; fst := Type);
 
   pp2 : Pair (B := Unit; A := Type) :=
-    mkPair (fst := Type) unit;
+    mkPair (fst := Type) (unit);
```

Comment:
Why this change?

Reply:
To catch more errors. When it begins with `(` it can still be a range, you see it's not only later.